### PR TITLE
added releasing checklist

### PR DESCRIPTION
**Pull request by borgmanJeremy**

### CODE REVIEW

File: docs/Releasing.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- [ ] Update changelog at data/debian/changelog
+- [ ] Update changelog at data/rpm/flameshot.spec
+- [ ] Update docs/appdata/flameshot.metainfo.xml
+
+These are the steps for actually making the release
+- [ ] Download all binaries from CI run started from PR related to code changes shown above
+- [ ] Create sha256 for each binary and compare against sha256 shown in the CI to verify there was no corruption or inserted malware.
+- [ ] Create a new "New Release" in githhub and explain changes in release notes
+- [ ] Upload all binaries and sha's
+- [ ] Update change log on [website](https://github.com/flameshot-org/flameshot-org.github.io/) data/changelog.md
```
Comment (ZetaoYang) [inline docs/Releasing.md:15]: And update version in https://github.com/flameshot-org/flameshot-org.github.io/blob/master/_coverpage.md

File: docs/Releasing.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Checklist for making a new release
+
+These are the code changes that need to take place
+- [ ] Create and push git tag
+- [ ] Update version in CMakeLists.txt
+- [ ] Update changelog at data/debian/changelog
```
Comment (ZetaoYang) [inline docs/Releasing.md:6]: also need update version at /data/debian/control

File: docs/Releasing.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- [ ] Update changelog at data/rpm/flameshot.spec
```
Comment (ZetaoYang) [inline docs/Releasing.md:7]: also need update version at /data/rpm/flameshot.spec

**Comment by hosiet** [comment]
Other parts look okay to me.

File: docs/Releasing.md (hunk 3 above)
Comment (borgmanJeremy) [reply docs/Releasing.md]: Thanks, i updated those comments, ill remove the version in the next release.

**Review by ZetaoYang — APPROVED** [review]

**Pull request merged**